eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -92,6 +92,7 @@ steps:
         DCP_DIAGNOSTICS_LOG_LEVEL: debug
         DCP_DIAGNOSTICS_LOG_FOLDER: $(Build.ArtifactStagingDirectory)/artifacts/log/dcp
         DCP_PRESERVE_EXECUTABLE_LOGS: 1
+        DOTNET_ROOT: $(Build.SourcesDirectory)/.dotnet
```

Comment:
Why is this needed? The build, and dotnet scripts should be able to pick the correct dotnet to use. `Build.SourcesDirectory)/.dotnet` will not be available if the required version is already installed system-wide.

Reply:
@radical The tests are run now as a separate executable which won't be aware of the correct .NET installation.

Reply:
Would this be a problem when this path does not exist? What about local runs?

Reply:
This is CI-only configuration. It doesn't affect local runs